ui/components/multichain/connected-site-permissions-modal/connected-site-permissions-modal.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { StoryFn, Meta } from '@storybook/react';
+import { ConnectedSitePermissionsModal } from '.';
+
+interface ConnectedSitePermissionsModalProps {
+  onClose: () => void;
+  siteName: string;
+  siteIcon: string;
+}
+
+export default {
+  title: 'Components/Multichain/ConnectedSitePermissionsModal',
+  component: ConnectedSitePermissionsModal,
+  argTypes: {
+    onClose: { control: 'function', action: 'onClose' },
```

Comment:
I don't think you need the function control here if you have the action. It assumes the type is a funciton
```suggestion
    onClose: { action: 'onClose' },
```